### ISSUE TITLE
classeme features

### ISSUE THREAD

**Issue by Dawn-LX** [Dawn-LX/VidSGG-BIG#3]
refer to https://github.com/Dawn-LX/VidSGG-BIG/issues/2#issuecomment-1122197607

**Comment by guanw-pku** [comment]
Thanks for these scripts.

I changed the `load_dir` to the parent directory of these `VidORtrain_freq1_m60s0.3_partxx` files to run `extract_logits.py`, while I got the assertion error. 

The length of res_path_list is 7006 not 7000. Besides, the assertion of` box_info` length return error as well. The length of the first box_info is 12 and the dimension of features is 1024.

There is the same issue (assertion error) in the `extract_logits_val.py`. The length of `res_path_list` is 842 not 835.

I also found that the `load_dir` in the original `extract_logits_val.py` is `.../nms0.5_miss30_score0.3/...`, while mine directory name is m60s0.3 (miss60_score0.3).

Could you please help me with these issues or release the generated classeme files?

Thank you!

**Comment by Dawn-LX** [comment]
> Thanks for these scripts.
> 
> I changed the `load_dir` to the parent directory of these `VidORtrain_freq1_m60s0.3_partxx` files to run `extract_logits.py`, while I got the assertion error.
> 
> The length of res_path_list is 7006 not 7000. Besides, the assertion of` box_info` length return error as well. The length of the first box_info is 12 and the dimension of features is 1024.
> 
> There is the same issue (assertion error) in the `extract_logits_val.py`. The length of `res_path_list` is 842 not 835.
> 
> I also found that the `load_dir` in the original `extract_logits_val.py` is `.../nms0.5_miss30_score0.3/...`, while mine directory name is m60s0.3 (miss60_score0.3).
> 
> Could you please help me with these issues or release the generated classeme files?
> 
> Thank you!

The classeme files is too large (i save them for every frame every box).

and, the 1024-dim feature is the RoI-pooled feature from MEGA, I did not release it, it's to large

`nms0.5_miss30_score0.3` is an ablation study, it not works well and not included in the paper. it has been deprecated

**Comment by Dawn-LX** [comment]
@guanw-pku 

In fact, you can use the "hard" classeme feature, as i mentioned above.
empirically the performance is similar.

Again, you can directly download the pre-prepared cached data, [here](https://github.com/Dawn-LX/VidSGG-BIG#tracklet-data-for-vidor)

**Comment by guanw-pku** [comment]
Thank you!

**Comment by MinamiKotoka** [comment]
> @guanw-pku
> 
> In fact, you can use the "hard" classeme feature, as i mentioned above. empirically the performance is similar.
> 
> Again, you can directly download the pre-prepared cached data, [here](https://github.com/Dawn-LX/VidSGG-BIG#tracklet-data-for-vidor)

Hi, I run `extract_logits.py` and want to obtain `VidORtrain_freq1_logits` to generate `classme` file of VidOR. But the errors are the same as above. So how can I get  `VidORtrain_freq1_logits` ?
You say 'hard' classme feature, but it is for VidVRD. So what is 'hard' classme feature for VidOR? And how can I obtain it?
Thank you very much!

**Comment by MinamiKotoka** [comment]
> @guanw-pku
> 
> In fact, you can use the "hard" classeme feature, as i mentioned above. empirically the performance is similar.
> 
> Again, you can directly download the pre-prepared cached data, [here](https://github.com/Dawn-LX/VidSGG-BIG#tracklet-data-for-vidor)

Hi, I run `extract_logits.py` and want to obtain `VidORtrain_freq1_logits` to generate `classme` file of VidOR. But the errors are the same as above. So how can I get  `VidORtrain_freq1_logits` ?
You say 'hard' classme feature, but it is for VidVRD. So what is 'hard' classme feature for VidOR? And how can I obtain it?
Thank you very much!

**Comment by MinamiKotoka** [comment]
> refer to [#2 (comment)](https://github.com/Dawn-LX/VidSGG-BIG/issues/2#issuecomment-1122197607)

Hi, I run `extract_logits.py` and want to obtain `VidORtrain_freq1_logits` to generate `classme` file of VidOR. But the errors are the same as above. So how can I get  `VidORtrain_freq1_logits` ?
You say 'hard' classme feature, but it is for VidVRD. So what is 'hard' classme feature for VidOR? And how can I obtain it?
Thank you very much!
